### PR TITLE
WIP: Spike for align and distribute

### PR DESCRIPTION
**Pull request by johnblackspear**
This is pretty scrappy - but it does spike out some basic alignment / and distribution. Horizontal distribution is skipped until we discuss this a little more :)

It does not yet take any subtleties into account and will move stuff pretty randomly since I don't filter for non-translatable or certain child shapes. 

Relates to #1098

### CODE REVIEW

File: src/components/nodes/utilities/AlignShapes.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+// TODO: this needs to be associated with the multi-selection box logic. We can
+// also pre-emptively recalc this everytime we add / remove items from a
+// multiselect
+export function getBoundingBox(shapes) {
+  const left = (shape) => shape.position().x;
+  const right = (shape) => shape.position().x + shape.size().width;
+  const top = (shape) => shape.position().y;
+  const bottom = (shape) => shape.position().y + shape.size().height;
+
+  const minX = minimum(shapes, left);
```
Comment (chipit24) [inline src/components/nodes/utilities/AlignShapes.js]: This is a cool helper. I wonder if using https://lodash.com/docs/4.17.15#minBy or `Math.min(shapes.map(left))` would simplify things.

File: src/components/modeler/setUpSelectionBox.js [hunk 2]
```diff
@@ -35,6 +44,32 @@ export default function setUpSelectionBox(setCursor, resetCursor, paperManager,
   let selectionBox;
 
   function shiftKeyDownListener(event) {
+
+    const highlightedShapes = store.getters.highlightedShapes;
```
Comment (chipit24) [inline src/components/modeler/setUpSelectionBox.js]: This is great for testing but I think in final-form they'd work nicely under icons in the toolbar.

Comment (johnblackspear) [reply src/components/modeler/setUpSelectionBox.js]: Yeah this needs to never go near actual humans :D

File: src/components/nodes/utilities/AlignShapes.js (hunk 1 above)
Comment (johnblackspear) [reply src/components/nodes/utilities/AlignShapes.js]: Super, thanks, that's better :)

**Comment by johnblackspear** [comment]
No longer needed as we've done the exploration for basic functionality that we wanted to

**Pull request closed without merging**